### ISSUE TITLE
Update technobabble in readme.md

### ISSUE THREAD

**Issue by Tricky1975** [PhantasarProductions/cynthia#75]

**Comment by Tricky1975** [comment]
This has already been taken care off, and should close automatically once the "Scyndi" branch is merged into this one.
See [this pull request](https://codeberg.org/PhantasarProductions/cynthia/pulls/1)
